packages/scandipwa/src/route/ProductPage/ProductPage.container.js

```diff
@@ -495,7 +495,7 @@ export class ProductPageContainer extends PureComponent {
         const currentVariantIndex = this.getConfigurableVariantIndex(variants);
         const variant = variants && variants[currentVariantIndex];
 
-        if (variants) {
+        if (variants?.length > 0) {
```

Comment:
You could have just used `if (variants?.length) { ... }`
But this option is fine as well.